perfkitbenchmarker/pkb.py

```diff
@@ -195,35 +258,46 @@ def RunBenchmark(benchmark, collector, sequence_number, total_benchmarks):
         logging.exception('Prerequisite check failed for %s', benchmark_name)
         raise
 
-    start_time = time.time()
+    # Initialize timed interval objects.
+    end_to_end_interval = timed_interval.TimedInterval('End to End')
+    prepare_phase_interval = timed_interval.TimedInterval('Prepare Phase')
+    bm_prepare_interval = timed_interval.TimedInterval('Benchmark Prepare')
+    run_phase_interval = timed_interval.TimedInterval('Run Phase')
+    bm_run_interval = timed_interval.TimedInterval('Benchmark Run')
+    cleanup_phase_interval = timed_interval.TimedInterval('Cleanup Phase')
+    bm_cleanup_interval = timed_interval.TimedInterval('Benchmark Cleanup')
```

Comment:
As above, recommend separating these into "Resource Teardown" and "Benchmark Cleanup".